README.md

```diff
@@ -13,5 +13,101 @@ Most of performed methods are logged using NLog, so you can easily see a history
 
 We use interfaces where is possible, so you can implement your own version of target interface with no need to rewrite other classes.
 
+### Quick start
+
+1. To start work with this package, simply add the nuget dependency Aquality.WinAppDriver to your project.
+
+2. Configure path to your application executable at settings.json:
+ - Create a `Resources` folder in your project and copy [settings.json](Aquality.WinAppDriver/src/Aquality.WinAppDriver/Resources/settings.json) into it. 
+ - Select `Copy to Output Directory`: `Copy always` option at settings.json file properties.
+ - Open settings.json and find `applicationPath` option under the `driverSettings` section. Replace the value with the path to your app.
+
+3. Ensure that [WinAppDriver](https://github.com/microsoft/WinAppDriver) is set up at your machine where the code would be executed.
+
+4. Start WinAppDriver service before your code execution. There are several options to do this:
+ - Start WinAppDriver.exe via external tool or command (e.g. manually with cmd/PS from the installation directory; via [task for you CI](https://marketplace.visualstudio.com/items?itemName=WinAppDriver.winappdriver-pipelines-task) )
+ - Start WinAppDriver automatically via AppiumLocalService - this option is integrated in our package, just set `"isRemote"` to `false` at your settings.json. This option requires specific version of node.js to be preinstalled on your machine (Please read more [here](http://appium.io/docs/en/contributing-to-appium/appium-from-source/#nodejs) and [here](https://github.com/appium/appium-dotnet-driver/wiki/How-to-start-an-AppiumDriver-locally)).
+ - Start WinAppDriver.exe manually calling the method `AqualityServices.WinAppDriverLauncher.StartWinAppDriverIfRequired();` from your code.
+ 
+5. (optional) Launch an application directly by calling `AqualityServices.Application.Launch();`. 
+
+> Note: If you don't start an Application directly, it would be started with the first call of any Aquality service or class requiring to interact with the Application.
+
+6. That's it! Now you are able work with Application via AqualityServices or via element services.
+```csharp
+
+ AqualityServices.Application.MouseActions.Click();
+ AqualityServices.Application.KeyboardActions.SendKeysWithKeyHold("n", ModifierKey.Control);
+```
+
+7. To interact with Application's windows, forms and elements, we recommend to follow the PageObjects pattern. This approach is fully integrated into our package.
```

Comment:
spellchecker shows to use 'we recommend following the PageObjects pattern'

Reply:
Thanks, fixed